README.md

```diff
@@ -1,11 +1,12 @@
 # GCP SSFW Deployer
 
-Internal package. Designed to automated deploying SSFW in GCP for stress testing.
+Internal package. Designed to automated deploying scalable ecommerce platform [Shopsys Framework (SSFW)](https://github.com/shopsys/shopsys) into [Google Cloud Platform (GCP)](https://cloud.google.com/gcp/) for performance and stress tests.
 
-Scripts for easy managing GCP.
-Creates a single virtual machine and run SSFW via docker-compose.
-Huge amount of data are loaded into in and production environment is set.
-Whole instance might be easily deleted as well.  
+There are scripts for easy managing GCP. Those scripts enables you to:
+
+* create a single virtual machine and run SSFW via docker-compose
+* load huge amount of data are loaded into SSFW and set production environment
```

Comment:
This bullet bellongs into one above. Performance data and production switch is done everytime you are deploying.